widget/embedded/src/pages/ConfirmSwapPage.tsx

```diff
@@ -326,8 +329,10 @@ export function ConfirmSwapPage() {
               type="primary"
               size="large"
               fullWidth
-              loading={fetchingConfirmationQuote}
-              disabled={!!confirmSwapResult.error || !isActiveTab}
+              loading={fetchingConfirmationQuote || isConfirming}
+              disabled={
+                !!confirmSwapResult.error || !isActiveTab || isConfirming
```

Comment:
Isn't adding it to the `loading` prop sufficient? When still `loading` is true, `onClick` does not trigger (based on `Button` component codes).

Reply:
fixed